app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -196,6 +196,62 @@
   border-bottom: 2px solid #aaa;
 }
 
+.panel-boundary-bottom {
+  padding-top: 5px;
```

Comment:
Properties should be ordered border-bottom, margin-left, margin-right, padding-top